documentation/source/overview/studies.rst

```diff
@@ -5,100 +5,118 @@ Studies using SCT
--  `Meyer et al. Optimized cervical spinal cord perfusion MRI after traumatic injury in the rat. J. of Cerebral Blood Flow & Metabolism, 2021 <https://journals.sagepub.com/doi/10.1177/0271678X20982396>`_
--  `Solanes et al. 3D patient-specific spinal cord computational model for SCS management: potential clinical applications. Journal of Neural Engineering, 2021 <https://pubmed.ncbi.nlm.nih.gov/33556926/>`_
--  `Johnson et al. Changes in White Matter of the Cervical Spinal Cord after a Single Season of Collegiate Football. Neurotrauma Reports, 2021 <https://www.liebertpub.com/doi/10.1089/neur.2020.0035>`_
--  `Ost et al. Spinal Cord Morphology in Degenerative Cervical Myelopathy Patients; Assessing Key Morphological Characteristics Using Machine Vision Tools. Journal of Clinical Medicine, 2021 <https://www.mdpi.com/2077-0383/10/4/892>`_
--  `Lee et al. Diffusion‐prepared fast spin echo for artifact‐free spinal cord imaging. Magnetic Resonance in Medicine, 2021 <https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28751>`_
--  `Tinnermann et al. Observation of others’ painful heat stimulation involves responses in the spinal cord. Science Advances, 2021 <https://pubmed.ncbi.nlm.nih.gov/33789899/>`_
--  `Zhang et al. Automatic spinal cord segmentation from axial-view MRI slices using CNN with grayscale regularized active contour propagation. Computers in Biology and Medicine, 2021 <https://pubmed.ncbi.nlm.nih.gov/33780869/>`_
--  `Savini et al. Pilot Study on Quantitative Cervical Cord and Muscular MRI in Spinal Muscular Atrophy: Promising Biomarkers of Disease Evolution and Treatment? Front Neurol, 2021 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8039452/>`_
--  `Vallotton et al. Tracking white and grey matter degeneration along the spinal cord axis in degenerative cervical myelopathy. J Neurotrauma. 2021 <https://pubmed.ncbi.nlm.nih.gov/34238034/>`_
--  `Shahrampour et al. Atlas-Based Quantification of DTI Measures in a Typically Developing Pediatric Spinal Cord. American Journal of Neuroradiology. 2021 <https://pubmed.ncbi.nlm.nih.gov/34326104/>`_
--  `Adibi et al. Quantitative Magnetic Resonance Imaging Analysis of Early Markers of Upper Cervical Cord Atrophy in Multiple Sclerosis and Neuromyelitis Optica Spectrum Disorder. Mult Scler Int. 2021 <https://pubmed.ncbi.nlm.nih.gov/34306756/>`_
--  `Martucci et al. Spinal Cord Resting State Activity in Individuals With Fibromyalgia Who Take Opioids. Front. Neurol. 2021 <https://www.frontiersin.org/articles/10.3389/fneur.2021.694271/full>`_
--  `David et al. Longitudinal changes of spinal cord grey and white matter following spinal cord injury. Journal of Neurology, Neurosurgery & Psychiatry. 2021 <https://jnnp.bmj.com/content/early/2021/08/11/jnnp-2021-326337.info>`_
+- **2021**
```

Comment:
Hmm... the date headers aren't showing up in the [top right-hand table of contents](https://spinalcordtoolbox--3502.org.readthedocs.build/en/3502/overview/studies.html) on the page. 

If you do this for each one:

```suggestion
2021
----
```

Then they'll be formatted as headers. (In RST, section headers are formatted using [different underline symbols](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections).)